scripts/build-installer-win64.bat

```diff
@@ -0,0 +1,102 @@
+rem DEPENDENCIES:
+rem   1. Node.js ('npm' binary in PATH)
+rem   2. 7zip    ('7z'  binary in PATH)
+
+set DEFAULT_CARDANO_BRANCH=cardano-sl-0.4
```

Comment:
@jmitchell, ok, so:
1. making `appveyor.yml` and `windows-build-fresh-daedalus.bat` specify the version
2. making the second argument (cardano version) mandatory in `scripts/build-installer-win64.bat`